js/libs/form.js

```diff
@@ -211,43 +211,13 @@ Fliplet.Widget.instance('form-builder', function(data) {
               }
               break;
 
-            case 'flCheckbox':
```

Comment:
You mentioned in the description:

> **Problem**: The data for the checkbox and radio button options was not generated in the checkbox and radio button components and this caused an error with the display of the selected options. The signature was not displayed for editing because there was no functionality to display it.
> **Solution**: The settings of the selected options have been transferred to the checkbox component and the radio button component. The preview is implemented for the signature component

I don't understand why the data wasn't being generated, and there why the problem must be solved by transferring the option setting to the component. I assume this worked at one point when the feature was initially built.

@OleksandrLavrynenko You may want to address this first before tackling the other changes.